templates/navigation_rustdoc.hbs

```diff
@@ -3,7 +3,7 @@
         <div class="pure-menu pure-menu-horizontal">
           <form action="/releases/search" method="GET" class="landing-search-form-nav">
             {{#unless varsb.show_search_form}}
-            <input class="search-input-nav" name="query" tabindex="-1" type="text" placeholder="Find crate"{{#if varss.search_query}} value="{{varss.search_query}}"{{/if}}>
+            <input class="search-input-nav" name="query" aria-label="Find crate by search query" tabindex="-1" type="text" placeholder="Find crate"{{#if varss.search_query}} value="{{varss.search_query}}"{{/if}}>
```

Comment:
The sentence here doesn't sound very nice... What about "Input to search crate"?

Reply:
Users should already know this is an input, based on the tag. The label should focus on what a user should expect to achieve by submitting the form with this field filled.